problemsets-mat223/modules/module6-exercises.tex

```diff
@@ -1,17 +1,86 @@
 \begin{exercises}
+	% Topics
+	% formal + intuitive definition of subspace
+	% relationship between subspace and spans
+	% prove a set is a subspace
+	% basis + dimension of a subspace
 	\begin{problist}
-		\prob  Example problem with parts
+
+		\prob If the following is a subspace, prove it. Otherwise, explain why not.
```

Comment:
-> "For each of the following sets, prove whether or not it is a subpace."

Note that "the following" is not a subspace. "The following" are questions...